docs/templates/plugins.rst.j2

```diff
@@ -76,6 +69,58 @@
             {{ raise_error(error_message) }}
         {% endif %}
     {% endif %}
+
+    Environment variable: ``{{ envvar }}``
+
+    In plan metadata:
+
+    .. code-block:: yaml
+
+       # how: {{ plugin_id }}
+       # ...
+
+    {% if metadata.metavar %}
+       {{ option }}: {{ metadata.metavar }}
+
+        {% for example in metadata.help_example_values %}
+       {{ option }}: {{ example }}
+        {% endfor %}
+    {% elif metadata.default is boolean %}
+       {{ option }}: true|false
+    {% else %}
+       {{ option }}:
+    {% endif %}
+
+    On command-line:
+
+    .. code-block:: shell
+
+       # tmt run ... {{ STEP }} --how {{ plugin_id }} ...
+
+    {% if metadata.metavar %}
+       {{ plugin_id }} --{{ option }} {{ metadata.metavar | shell_quote }} ...
```

Comment:
```suggestion
       --{{ option }} {{ metadata.metavar | shell_quote }} ...
```

I would drop the plugin name here as it looks like it is a (sub)command, but it's not, it needs the `step --how` before it. I'd say the option would be just enough and is easy for quick copy-pasting.

Reply:
Ok, I can do that.

Reply:
0cc11fd1